tests/db/migration_tests/test_v202301110_to_v202301160.py

```diff
@@ -0,0 +1,44 @@
+"""
+    Copyright 2023 Inmanta
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+        http://www.apache.org/licenses/LICENSE-2.0
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+    Contact: code@inmanta.com
+"""
+import os
+from collections import abc
+
+import pytest
+
+
+@pytest.mark.db_restore_dump(os.path.join(os.path.dirname(__file__), "dumps/v202301160.sql"))
```

Comment:
```suggestion
@pytest.mark.db_restore_dump(os.path.join(os.path.dirname(__file__), "dumps/v202301100.sql"))
```

Your dump seems to be for the old schema. Did you generate it before you wrote the migration script?

Reply:
I generated it on the master branch, as it is what I understood should be done from the adr. But I don't understand why I should use v202301100.sql? I thought the Idea was to create the dump before the changes and to migrate from that dump in the migration test?